dashboard/package.json

```diff
@@ -29,6 +29,7 @@
     "framer-motion": "^6.5.1",
     "lodash": "^4.17.21",
     "next": "^12.0.9",
+    "protoc-gen-ts": "^0.8.5",
```

Comment:
The proto generator should be part of `ts-proto`? Do we really need to add this dependency?

Reply:
Also if you need to add it, move to devDependency.

Reply:
done